### PR TITLE
add process proposal and issue templates

### PR DESCRIPTION
**Pull request by vtbassmatt**
Process proposal

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+At a high level, the process looks like this:
+
+1.  [Create an issue](/issues/new?template=tools.md) in this repo. Be sure to fill out the issue template to help us route your request quickly.
+2.  Participate in the discussion via comments.
+Our images have real constraints, especially disk space.
+We have to balance the value (broad adoption of that tool) vs. cost (disk space and on-going maintenance) proposition.
+3.  Optionally, create a PR.
+For a new tool, it's probably not worth your time to create the PR until some discussion happens.
+However, if you're just bumping a minor or patch version, feel free to create a PR and reference the issue.
+4. After we approve a new or updated tool, someone from our team will make the change.
```
Comment (DavidStaheli) [inline .github/CONTRIBUTING.md]: "someone from our team will make the change or merge your PR."

File: .github/ISSUE_TEMPLATE/tools.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Help us route
+
+This is the template for requesting new or updated tools on the hosted images.
+If you intended to file a bug, please switch to the Bugs template.
+
+# If this is a request for a new tool...
```
Comment (DavidStaheli) [inline .github/ISSUE_TEMPLATE/tools.md]: "If this is a request for a new tool or tool version..."

File: .github/ISSUE_TEMPLATE/tools.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- [ ] Python
+- [ ] Java
+- [ ] Ruby
+- [ ] PHP
+- [ ] Rust
+- [ ] Database servers or clients
+- [ ] Testing and code coverage
+- [ ] General purpose tools
+- [ ] Other
+
```
Comment (DavidStaheli) [inline .github/ISSUE_TEMPLATE/tools.md]: Maybe we should add official checkboxes for "Container tools" and Scala.  And can we alphabetize these except for keeping "Other" at the bottom?

Comment (vtbassmatt) [reply .github/ISSUE_TEMPLATE/tools.md]: That makes sense.

**Comment by vtbassmatt** [comment]
I haven't heard any more objections. Will merge this later today unless someone shouts.

**Pull request merged**